src/main/webapp/app/course/manage/overview/course-management-overview-statistics-dto.model.ts

```diff
@@ -0,0 +1,9 @@
+import { CourseManagementOverviewExerciseStatisticsDTO } from 'app/course/manage/overview/course-management-overview-exercise-statistics-dto.model';
+
+export class CourseManagementOverviewStatisticsDto {
+    public courseId: number;
```

Comment:
are you sure those values always exist? is this 100% guaranteed?

Reply:
Yes, we always explicitly set this on the server side.